src/ert/shared/share/ert/workflows/jobs/internal-gui/scripts/csv_export.py

```diff
@@ -224,7 +221,7 @@ def getArguments(self, parent, ert_config, storage):
         raise CancelPluginException("User cancelled!")
 
     def get_context_value(self, name, default):
-        context = self.ert_config.substitution_list
+        context = self.ert().substitution_list
```

Comment:
Why is this using the deprecated property? :thinking:

Reply:
Good point, fixed!